build/.goreleaser.yml

```diff
@@ -60,10 +60,17 @@ nfpms:
         type: config
 
     overrides:
+      deb:
+        dependencies:
+          - newrelic-infra (>= 1.20.0)
+          - nrjmx (>= 2.0.1)
```

Comment:
If we are overriding both `deb` and `rpm`, couldn't we just place this under the general `nfpms.dependencies`?

Reply:
we'll have to check if on rpm works with brackets. '(>= x.y.z)' vs `>= x.y.z`. It looks like for newrelic-infra dependency works.